lib/mongo/uri.rb

```diff
@@ -658,14 +674,14 @@ def journal(value)
     end
 
     # Parses the ssl_verify value. Note that this will be the inverse of the value of
-    # tlsAllowInvalidCertificates (if present).
+    # tlsInsecure (if present).
     #
-    # @param value [ String ] The tlsAllowInvalidCertificates value.
+    # @param value [ String ] The tlsInsecure value.
     #
     # @return [ true | false | nil ] The ssl_verify value parsed out, otherwise nil (and a warning
     #   will be logged).
     def ssl_verify(value)
-      b = bool('tlsAllowInvalidCertificates', value)
+      b = bool('tlsInsecure', value)
```

Comment:
Am I reading this correctly that setting `ssl_verify: 'bogus'` would produce an error message referencing `tlsInsecure`?

Reply:
`ssl_verify` cannot be used from the URI at all, so it would just give a warning for an unknown URI option